packages/client/hmi-client/src/components/model/petrinet/tera-model-description.vue

```diff
@@ -193,7 +193,7 @@ const emit = defineEmits(['update-model', 'fetch-model', 'update-configuration',
 
 const teraModelDiagramRef = ref();
 
-const card = computed(() => {
+const card: Card | any | null = computed(() => {
```

Comment:
This mixing of concrete types and any doesn't make sense to me. We either type it properly
- Card | null | soemthing, or we give up and just do
- `any` without additional typing information